benchmarks/conftest.py

```diff
@@ -0,0 +1,152 @@
+from attr import dataclass
+import json
+import pytest
+
+from ariadne import QueryType, gql, make_executable_schema
+
+
+@dataclass
+class Person:
+    name: str
+    age: int
+
+
+@dataclass
+class User:
+    id: int
+    name: str
+    group: dict
+    avatar: list
```

Comment:
This should be `List[Avatar]` where `Avatar` is dataclass.

Reply:
Same as above